site/blocks/ExpandableParagraph.tsx

```diff
@@ -64,6 +64,10 @@ export const ExpandableParagraph = (
         <div className={cx("expandable-paragraph", className)}>
             <div
                 style={contentStyles}
+                // inert isn't supported in react@17
+                // prevents focus on elements that are not visible
+                // ideally would only apply to elements below the fold but that's hard
+                {...{ inert: isClosed ? "true" : undefined }}
```

Comment:
Is this really what we want? What problem do we solve? It also prevents users from selecting text within the element, and the text will be excluded from browser search, which seems too harsh.

Reply:
The issue I was trying to address here is that you can tab through links that are hidden, which renders poorly (see the video.)

I tried setting up a prop that would set `tabIndex="-1"` on any nested `span-link` elements but couldn't get it to work in the small amount of time I was budgeting for this PR so I went with this simpler solution instead.

I'm open to changing this though, if you think this change is an overall downgrade 🙂 

https://github.com/user-attachments/assets/7a850299-dd8b-4364-8e98-5204ee0677af

Reply:
Hmm, I'm not sure what's better. Let's go with what you did and revert/rework later, if necessary.